pom.xml

```diff
@@ -585,12 +585,27 @@
       </dependency>
       <dependency>
         <groupId>org.elasticsearch.plugin</groupId>
+        <artifactId>ingest-attachment</artifactId>
+        <version>${org.elasticsearch.version}</version>
+      </dependency>
+      <dependency>
+        <groupId>org.elasticsearch.plugin</groupId>
         <artifactId>mapper-attachments</artifactId>
         <version>${org.elasticsearch.version}</version>
       </dependency>
       <dependency>
+        <groupId>org.codelibs.elasticsearch.module</groupId>
```

Comment:
Why don't we do the same thing for this Maven dependency as we did for ingest-attachement and mapper-attachment? (I mean to upload it on our Nexus repositories)